src/helpers/json_to_predreq.py

```diff
@@ -36,4 +36,6 @@ def decode(request):
         task=request.model.task.lower(),
         featurizer=featurizers,
     )
+    if request.model["selectedFeatures"] != []:
```

Comment:
This won't work anymore, we have types now for the request, so now the request is not a dictionary but an object. Here's how we do it now:
```suggestion
    if len(request.model.selected_features) > 0:
```